packages/contracts/deploy/01_libs.ts

```diff
@@ -16,8 +16,12 @@ const func: DeployFunction = async (hre: HardhatRuntimeEnvironment) => {
     const { deploy } = deployments;
     const { deployer } = await getNamedAccounts();
 
-    // IoTeX does not support the deterministic deployment through the contract used by hardhat-deploy
-    const deterministicDeployment = network.name !== "iotex_testnet";
+    // IoTeX doesn't have support yet, see https://github.com/safe-global/safe-singleton-factory/issues/199
+    // Chiado is not working, see https://github.com/safe-global/safe-singleton-factory/issues/201
+    const nonDeterministicNetworks = ["iotex_testnet", "chiado"];
```

Comment:
I think we should move this to a file of its own, and import only a function like `supportsDeterministicDeployment` that takes the network name and returns a boolean. Otherwise, we have to maintain this list in multiple files.

Reply:
I thought about that, but it's a little tricky, as it's a deploy script. Every file inside the `deploy` directory is executed as a deploy script, so creating a new file there might create problems.
And creating a file inside `src/` also may not be a good idea, but we are packing the deploy script so other projects can "import" and execute them during a local execution.

I think as this is a temporary solution it's ok to be duplicated.